README.md

```diff
@@ -8,8 +8,9 @@ The main technical documentation for the BBC micro:bit is located at the [micro:
 The site will provide your information on how the micro:bit works, a reference-design based on the micro:bit, as well as a detailed overview of
 the schematic contained in this repository.
 
-* [Overview of the hardware](https://tech.microbit.org/hardware/).
-* [Walk-through of the schematic](https://tech.microbit.org/hardware/schematic/).
+* [Overview of the hardware](https://tech.microbit.org/hardware/)
+* [Walk-through of the schematic](https://tech.microbit.org/hardware/schematic/)
+* [V2 board test points](https://tech.microbit.org/hardware/schematic/)
```

Comment:
I've just updated the site to include an anchor directly to the test point map https://tech.microbit.org/hardware/schematic/#test-point-map 

Can we update the link here accordingly?

Reply:
Ah, I didn't know how to anchor it. Learn a new thing every day - thanks Mark. So to confirm the URL needs to change to [https://tech.microbit.org/hardware/schematic/#test-point-map] which will go directly to the location on the page?

Reply:
Yes. You can add this suggestion if you like
```suggestion
* [V2 board test points](https://tech.microbit.org/hardware/schematic/#test-point-map)
```